### PR TITLE
Update 04-http-create-post.md

### PR DESCRIPTION
**Pull request by pavanvora**
used API is /blog/posts to fetch unsigned create post transaction but actually it is /blog/post in x/blog/client/rest/rest.go file

### CODE REVIEW

**Review by fadeev — APPROVED** [review]
Thank you, @pavanvora!

**Comment by pavanvora** [comment]
> Thank you, @pavanvora!

Sorry to bother you but Is anything wrong with this PR? It's about to month still did not merge.

File: starport-blog/tutorial/04-http-create-post.md [hunk 1]
```diff
@@ -134,6 +134,7 @@ Next, we need to define the `createPost` function. It will take the value from t
 const createPost = () => {
   // Getting a post title value from text input
   const title = document.getElementById("input").value;
+  const body = document.getElementById("body").value;
```
Comment (xiaojian-zhou) [inline starport-blog/tutorial/04-http-create-post.md:137]: the title should also be modified to `getElementById("title")`

**Comment by lukitsbrian** [comment]
@pavanvora there are some conflicts with the current master branch, could you update your branch and re-request the review?

**Review by lukitsbrian — CHANGES_REQUESTED** [review]
@pavanvora there are some conflicts with the current master branch, could you update your PR and re-request the review?

File: starport-blog/tutorial/04-http-create-post.md (hunk 1 above)
Comment (pavanvora) [reply starport-blog/tutorial/04-http-create-post.md:137]: Thank you, @xiaojian-zhou. I'll do that.

**Comment by pavanvora** [comment]
Okay @lukitsbrian I'll have a look and update the PR.

File: starport-blog/tutorial/04-http-create-post.md (hunk 1 above)
Comment (pavanvora) [reply starport-blog/tutorial/04-http-create-post.md]: @xiaojian-zhou I've updated the PR with all the changes you asked. Can you please review that?

**Comment by pavanvora** [comment]
> @pavanvora there are some conflicts with the current master branch, could you update your PR and re-request the review?

I did resolve the conflicts and updated the PR. can you please review that?

**Review by marbar3778 — APPROVED** [review]

**Pull request merged**